test/e2e/storage/external/external.go

```diff
@@ -99,7 +100,10 @@ func (t testDriverParameter) loadDriverDefinition(filename string) (*driverDefin
 				"", // Default fsType
 			),
 		},
-		ClaimSize: "5Gi",
+		SupportedSizeRange: volume.SizeRange{
+			Min: "1Mi",
+			Max: "10Gi",
```

Comment:
The original size is 5Gi, so I guess this Min should be 5Gi and the Max should be empty as previous discussion.
Or do we have some reason for this combination of Min and Max? There are the same combination on the following.

Reply:
Thanks for pointing this out. Have reset to original size 5Gi